data/data/bootstrap/baremetal/files/etc/containers/systemd/image-customization.container

```diff
@@ -22,6 +22,7 @@ Environment="DEPLOY_INITRD=/shared/html/images/ironic-python-agent.initramfs"
 Environment="IRONIC_BASE_URL=${IRONIC_BASE_URL}"
 Environment="IRONIC_RAMDISK_SSH_KEY=${IRONIC_RAMDISK_SSH_KEY}"
 Environment="IRONIC_AGENT_IMAGE=${IRONIC_AGENT_IMAGE}"
+Environment="IRONIC_AGENT_PULL_SECRET=${IRONIC_AGENT_PULL_SECRET}"
```

Comment:
This is undoing the change in #5525. An equivalent change to https://github.com/openshift/image-customization-controller/pull/30 in the k8s i-c-c would allow us to keep doing what we were doing.

Reply:
https://github.com/openshift/image-customization-controller/pull/115

Reply:
Merged